test/e2e/e2e.go

```diff
@@ -40,3 +45,83 @@ func SetupWithNamespace(t *testing.T, namespace string) *test.Clients {
 	}
 	return clients
 }
+
+// knativeServingVerify verifies if the KnativeServing can reach the READY status.
+func knativeServingVerify(t *testing.T, clients *test.Clients, names test.ResourceNames) {
+	if _, err := resources.WaitKnativeServingReady(t, clients.KnativeServingAlphaClient, names.KnativeServing,
+		resources.IsKnativeServingReady); err != nil {
+		t.Fatalf("KnativeService %q failed to get to the READY status: %v", names.KnativeServing, err)
+	}
+
+}
+
+// deploymentRecreation verify whether all the deployments for knative serving are able to recreate, when they are deleted.
+func deploymentRecreation(t *testing.T, clients *test.Clients, names test.ResourceNames) {
```

Comment:
Do you anticipate us ever reusing these helpers? I feel like they belong next to the test in the appropriate file.

Reply:
I will move them into the test file.